src/server/api/controllers/orders.controller.js

```diff
@@ -0,0 +1,27 @@
+const knex = require('../../config/db');
+const HttpError = require('../lib/utils/http-error');
+
+const getOrders = async () => {
+  return knex('orders');
+};
+
+const getOrderById = async (id) => {
+  if (!id) {
+    throw new HttpError('Id should be a number', 400);
```

Comment:
You can change this error to the one that you show in Swagger documentation

Reply:
Done